diff --git a/include/pybind11/stl.h b/include/pybind11/stl.h
index 721bb66..5c55646 100644
--- a/include/pybind11/stl.h
+++ b/include/pybind11/stl.h
@@ -31,11 +31,11 @@
 #    define PYBIND11_HAS_OPTIONAL 1
 #  endif
 // std::experimental::optional (but not allowed in c++11 mode)
-#  if defined(PYBIND11_CPP14) && (__has_include(<experimental/optional>) && \
-                                 !__has_include(<optional>))
-#    include <experimental/optional>
-#    define PYBIND11_HAS_EXP_OPTIONAL 1
-#  endif
+// #  if defined(PYBIND11_CPP14) && (__has_include(<experimental/optional>) && \
+//                                  !__has_include(<optional>))
+// #    include <experimental/optional>
+// #    define PYBIND11_HAS_EXP_OPTIONAL 1
+// #  endif
 // std::variant
 #  if defined(PYBIND11_CPP17) && __has_include(<variant>)
 #    include <variant>
